tasks/7.txt

```diff
@@ -4,4 +4,15 @@ Create a query to return all employees and full name of person to whom this empl
 
 Full Name - title of courtesy with full name.
 Order data by first column.
-Reports To - Full name. If the employee does not report to anybody leave "-" in the column.
\ No newline at end of file
+Reports To - Full name. If the employee does not report to anybody leave "-" in the column.
+
+db.employees.aggregate([
+    {$lookup: {
+          from: "employees",
+          localField: "ReportsTo",
+          foreignField: "EmployeeID",
+          as: "ReportsToQ"
+        }},
+            {$unwind: "$ReportsToQ"},
+    {$project: {EmployeeID: 1, FullName: { $concat: [ "$FirstName", " ", "$LastName" ]}, ReportsTo: {$concat: ["$ReportsToQ.FirstName"," ", "$ReportsToQ.LastName"]}}}
+    ]);
```

Comment:
you missed employees with ReportsTo = null;
See unwind
https://docs.mongodb.com/manual/reference/operator/aggregation/unwind/